.github/workflows/dev.yml

```diff
@@ -30,10 +28,10 @@ jobs:
       - name: Run docker container
         run: docker run --name sporkfish-container -d -it sporkfish-image
       # -- Inspect code/tests --
-      - name: Check typing (mypy)
-        run: docker exec sporkfish-container mypy --install-types --non-interactive
       - name: Check tests (pytest)
         run: docker exec sporkfish-container pytest -sv
+      - name: Check typing (mypy)
+        run: docker exec sporkfish-container mypy --install-types --non-interactive
```

Comment:
discussed offline, happy with this approach - lets see how it works out. Very low cost to change, can always amend again.